src/lib/LBT/LBT.cpp

```diff
@@ -112,10 +151,11 @@ bool ICACHE_RAM_ATTR ChannelIsClear(void)
 
   int8_t rssiInst = Radio.GetRssiInst();
   Radio.SetTxIdleMode();
```

Comment:
I dont think this is required.  See comment below...

Reply:
I'll do some more testing over the next few days with this removed. I did wonder about that line when I was looking at the code 😄

Reply:
Removing this line causes the TX to fail in unexpected ways!
1. it will not connect to the RX in FLRC mode.
2. changing the rate causes the radio to lockup!
Removing it does not appear to have any negative effects on the RX side.

Reply:
Hmm... Ill have a look on the LA and see if its failing in an obvious way.